examples/rstudio/start_rstudio

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+#password=$(openssl rand -base64 16)
+# Just use a basic password but in practice we'd want to create a random one like above
+password=charliecloud
+export RSTUDIO_PASSWORD=${password}
+
+port=${1}
+if [[ -z ${port} ]]; then
+  echo "You must specify a port. Suggested range: 8000-9000"
+  exit 1
+fi
+
+RSTUDIO_AUTH="/bin/rstudio_auth"
+
+/usr/lib/rstudio-server/bin/rserver \
+  --www-port="${port}" \
+  --auth-none=0 \
+  --auth-pam-helper-path=${RSTUDIO_AUTH} \
```

Comment:
What are the implications of running an open Rstudio server during the test? Looks like it's listening to all interfaces too.